homeassistant/components/owntracks/manifest.json

```diff
@@ -4,7 +4,10 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/owntracks",
   "requirements": [
-    "PyNaCl==1.3.0"
+    "PyNaCl==1.3.0",
+    "paho-mqtt==1.5.0",
```

Comment:
These should be required by the mqtt integration already.

@balloob, correct?

Reply:
My setup complained about these missing dependencies. Unless mqtt integration is enabled by default, I think my setup does not have it enabled.

Reply:
There might be a bug somewhere. I'm not sure this change is the right approach. We should here what @balloob says.

Reply:
I have mqtt integration enabled, but got the same error reported in #28388 when I ran the config check.

```
starting version 3.2.8
[10:09:15] INFO: Don't worry, this temporary installation is not overwriting your current one.
[10:09:15] INFO: Installing Home Assistant: latest...
[10:09:15] INFO: Please be patient, this might take a few minutes...
WARNING: You are using pip version 19.2.2, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
[10:09:50] INFO: Installed Home Assistant 0.101.1
[10:09:50] INFO: Making a copy of your configuration for checking...
[10:10:00] INFO: Checking your configuration against this version...
[10:14:42] ERROR: The configuration check did not pass!
[10:14:42] ERROR: See the output below for more details.
INFO:homeassistant.util.package:Attempting install of colorlog==4.0.2
Testing configuration at /tmp/config
INFO:homeassistant.util.package:Attempting install of mutagen==1.42.0
INFO:homeassistant.util.package:Attempting install of aiohttp_cors==0.7.0
INFO:homeassistant.util.package:Attempting install of gTTS-token==1.1.3
INFO:homeassistant.util.package:Attempting install of libpyfoscam==1.0
INFO:homeassistant.util.package:Attempting install of ha-ffmpeg==2.0
INFO:homeassistant.util.package:Attempting install of ring_doorbell==0.2.3
INFO:homeassistant.util.package:Attempting install of distro==1.4.0
INFO:homeassistant.util.package:Attempting install of PyNaCl==1.3.0
INFO:homeassistant.util.package:Attempting install of aiofiles==0.4.0
INFO:homeassistant.util.package:Attempting install of aiogithubapi==0.4.2
INFO:homeassistant.util.package:Attempting install of backoff==1.8.0
INFO:homeassistant.util.package:Attempting install of integrationhelper==0.2.2
INFO:homeassistant.util.package:Attempting install of sqlalchemy==1.3.10
INFO:homeassistant.util.package:Attempting install of python-ecobee-api==0.1.4
INFO:homeassistant.util.package:Attempting install of pyfttt==0.3
INFO:homeassistant.util.package:Attempting install of homeassistant-pyozw==0.1.4
INFO:homeassistant.util.package:Attempting install of hbmqtt==0.9.5
INFO:homeassistant.util.package:Attempting install of paho-mqtt==1.4.0
INFO:homeassistant.util.package:Attempting install of google-api-python-client==1.6.4
INFO:homeassistant.util.package:Attempting install of httplib2==0.10.3
INFO:homeassistant.util.package:Attempting install of oauth2client==4.0.0
INFO:homeassistant.util.package:Attempting install of home-assistant-frontend==20191025.1
INFO:homeassistant.util.package:Attempting install of quantum-gateway==0.0.5
INFO:homeassistant.util.package:Attempting install of aioautomatic==0.6.5
INFO:homeassistant.util.package:Attempting install of python-forecastio==1.4.0
INFO:homeassistant.util.package:Attempting install of waqiasync==1.0.0
INFO:homeassistant.util.package:Attempting install of psutil==5.6.3
INFO:homeassistant.util.package:Attempting install of pyhaversion==3.1.0
INFO:homeassistant.util.package:Attempting install of python-twitch-client==0.6.0
INFO:homeassistant.util.package:Attempting install of python-nest==4.1.0
INFO:homeassistant.util.package:Attempting install of WazeRouteCalculator==0.10
INFO:homeassistant.util.package:Attempting install of deluge-client==1.7.1
INFO:homeassistant.util.package:Attempting install of plexapi==3.0.6
INFO:homeassistant.util.package:Attempting install of plexauth==0.0.5
INFO:homeassistant.util.package:Attempting install of plexwebsocket==0.0.3
INFO:homeassistant.util.package:Attempting install of bellows-homeassistant==0.10.0
INFO:homeassistant.util.package:Attempting install of zha-quirks==0.0.26
INFO:homeassistant.util.package:Attempting install of zigpy-deconz==0.6.0
INFO:homeassistant.util.package:Attempting install of zigpy-xbee-homeassistant==0.6.0
INFO:homeassistant.util.package:Attempting install of zigpy-zigate==0.5.0
INFO:homeassistant.util.package:Attempting install of hass-nabucasa==0.22
INFO:homeassistant.util.package:Attempting install of speedtest-cli==2.1.1
INFO:homeassistant.util.package:Attempting install of numpy==1.17.3
INFO:homeassistant.util.package:Attempting install of pyiqvia==0.2.1
INFO:homeassistant.util.package:Attempting install of pushbullet.py==0.11.0
INFO:homeassistant.util.package:Attempting install of aiowwlln==2.0.2
INFO:aiocache:aioredis not installed, RedisCache unavailable
INFO:aiocache:aiomcache not installed, Memcached unavailable
INFO:homeassistant.util.package:Attempting install of pyHS100==0.3.5
INFO:homeassistant.util.package:Attempting install of netdisco==2.6.0
INFO:homeassistant.util.package:Attempting install of ourgroceries==1.2.3
INFO:homeassistant.util.package:Attempting install of av==6.1.2
INFO:homeassistant.util.package:Attempting install of alexapy==1.3.2
INFO:homeassistant.util.package:Attempting install of pywemo==0.4.34
INFO:homeassistant.util.package:Attempting install of pynws==0.8.1
INFO:homeassistant.util.package:Attempting install of adb-shell==0.0.7
INFO:homeassistant.util.package:Attempting install of androidtv==0.0.32
INFO:homeassistant.util.package:Attempting install of gmusicapi>=11.0.4
INFO:homeassistant.util.package:Attempting install of lightify==1.0.7.2
INFO:homeassistant.util.package:Attempting install of pynanoleaf==0.0.5
INFO:homeassistant.util.package:Attempting install of aioharmony==0.1.13
Failed config
  General Errors: 
    - Component error: owntracks - No module named 'paho'

Successful config (partial)
```